src/Helper/Navigation/AbstractHelper.php

```diff
@@ -244,7 +244,7 @@ public function findActive($container, $minDepth = null, $maxDepth = -1)
      *
      * @param AbstractContainer|string|null $container
      * @return void
-     * @param-out AbstractContainer $container
+     * @param-out AbstractContainer|null $container
```

Comment:
Possibly a good idea to declare a parametrized `param-out`, to remove nullability scenarios from a call-site perspective

Reply:
Sadly it lokos not possible with PSALM: https://github.com/vimeo/psalm/issues/8176

Reply:
Then let's skip until possible, IMO